child_compassion/models/compassion_reservation.py

```diff
@@ -184,10 +184,6 @@ def send_reservation(self):
         messages = self.with_context(async_mode=False).handle_reservation()
         for i in range(0, len(messages)):
             if messages[i].state != 'success':
-                self[i].message_post(
-                    messages[i].failure_reason,
-                    _("Reservation failed")
-                )
```

Comment:
You can remove all the for loop which is not useful anymore. You can simply end the function with :
```
return self.with_context(async_mode=False).handle_reservation()
```